presto-docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -417,7 +417,7 @@ this decoder should be used.
 Date and Time Decoders
 ^^^^^^^^^^^^^^^^^^^^^^
 
-To convert values from JSON objects into Presto ``DATE``, ``TIME``, ``TIME WITH TIME ZONE`,
+To convert values from JSON objects into Presto ``DATE``, ``TIME``, ``TIME WITH TIME ZONE``,
```

Comment:
Recently @electrum (see https://github.com/prestosql/presto/commit/ada6d8a3f735de95ca0772e326d2d1927cd1cb0a) added a check to fail the build when there is only single backquote used. It would be nice to have something like that for this kind of bugs.